### ISSUE TITLE
No way to view image details (file name / EXIF)

### ISSUE THREAD

**Issue by flyingbobcat** [ubports/gallery-app#4]
There is no way to view the file name of an image. Neither can one see [EXIF](https://en.wikipedia.org/wiki/Exif) information like exposure time, shutter speed, ISO value etc.

There are OSS like [eog](https://wiki.gnome.org/Apps/EyeOfGnome), [shotwell](https://wiki.gnome.org/Apps/Shotwell) and [gimp](https://www.gimp.org/), which can display EXIF information.

**Comment by Flohack74** [comment]
I would suggest to overlay the filename already in the normal view and to add some menu entry to view further details (if they are available). Just had a hard time copying the right screenshot from the phone as I could not determine the filename of it ;) => had to copy ALL screenshots...

**Comment by TronFourtyTwo** [comment]
Yes. I am already working on a prototype of bottom edge with advanced informations and settings.

**Comment by TronFortyTwo** [comment]
See #84

**Comment by Flohack74** [comment]
Hmm send me a screenshot to approve ;)

**Comment by TronFortyTwo** [comment]
![img_20181031_221843_720](https://user-images.githubusercontent.com/18293963/47819498-fa3e9280-dd5a-11e8-9006-7d8bd43c96f4.jpg)
Here it is! :D

**Comment by StefWe** [comment]
first this need to be merged? https://github.com/ubports/gallery-app/pull/84

**Comment by Flohack74** [comment]
Oh this on up there was smth else
